src/test/java/com/ays/institution/controller/InstitutionControllerSystemTest.java

```diff
@@ -0,0 +1,85 @@
+
+import java.util.Arrays;
+import java.util.List;
+
+class InstitutionControllerSystemTest extends AbstractSystemTest {
+
+
+    private final InstitutionToInstitutionResponseMapper institutionToInstitutionResponseMapper = InstitutionToInstitutionResponseMapper.initialize();
+
+
+    private static final String BASE_PATH = "/api/v1/institutions";
+
+    @Test
+    void givenNothing_whenInstitutionStatusActive_thenReturnListInstitutionResponse() throws Exception {
+
+
+        String mockAccessToken = superAdminToken.getAccessToken();
+
+        Institution institution1 = new InstitutionBuilder().withValidFields().build();
+        Institution institution2 = new InstitutionBuilder().withValidFields().build();
+
+
+        List<Institution> mockInstitutions = Arrays.asList(
+                institution1,
+                institution2
+        );
+
+
+        // then
+        List<InstitutionResponse> mockInstitutionResponseList = institutionToInstitutionResponseMapper.map(mockInstitutions);
+        AysResponse<List<InstitutionResponse>> mockAysResponse = AysResponse.successOf(mockInstitutionResponseList);
+
+        mockMvc.perform(AysMockMvcRequestBuilders
+                        .get(BASE_PATH.concat("/summary"), mockAccessToken))
+                .andDo(MockMvcResultHandlers.print())
+                .andExpect(AysMockResultMatchersBuilders.status().isOk())
+                .andExpect(AysMockResultMatchersBuilders.time()
+                        .isNotEmpty())
+                .andExpect(AysMockResultMatchersBuilders.httpStatus()
+                        .value(mockAysResponse.getHttpStatus().getReasonPhrase()))
+                .andExpect(AysMockResultMatchersBuilders.isSuccess()
+                        .value(mockAysResponse.getIsSuccess()))
+                .andExpect(AysMockResultMatchersBuilders.response()
+                        .isNotEmpty());
+
+
+    }
+
+    @Test
+    void givenNothing_whenUserUnauthorizedForSummary_thenReturnAccessDeniedException() throws Exception {
+
+        // Given
+        String mockAccessToken = userTokenOne.getAccessToken();
+
+
+        MockHttpServletRequestBuilder mockHttpServletRequestBuilder = AysMockMvcRequestBuilders
+                .get(BASE_PATH.concat("/summary"), mockAccessToken);
+
+        AysResponse<AysError> mockResponse = AysResponseBuilder.FORBIDDEN;
+        mockMvc.perform(mockHttpServletRequestBuilder)
+                .andDo(MockMvcResultHandlers.print())
+                .andExpect(AysMockResultMatchersBuilders.status().isForbidden())
+                .andExpect(AysMockResultMatchersBuilders.time().isNotEmpty())
+                .andExpect(AysMockResultMatchersBuilders.httpStatus().value(mockResponse.getHttpStatus().name()))
+                .andExpect(AysMockResultMatchersBuilders.isSuccess().value(mockResponse.getIsSuccess()))
+                .andExpect(AysMockResultMatchersBuilders.response().doesNotExist());
+
+    }
+
+}
```

Comment:
Bu sınıfın en altında bir tane fazladan boş satır olması gerekiyor. GitHub da bu konuda uyarıyor
![image](https://github.com/afet-yonetim-sistemi/ays-be/assets/61833677/9558cc04-1af2-4b20-a8b0-87dceb5c8943)

Reply:
Bu kısmı revise ettim